scarb/scarblib/cairo_test_asserts/Scarb.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "cairo_test_macros"
```

Comment:
Maybe we can name it `assert_macros`? This can be used not only in test, also there is nothing more than `assert_*!` here

Reply:
+1